packages/types/src/interfaces/eth/rpc.ts

```diff
@@ -26,9 +26,24 @@ const netRpc: DefinitionsRpc = {
     type: 'String'
   }
 };
+const web3Rpc: DefinitionsRpc = {
+  clientVersion: {
+    aliasSection: 'web3',
+    description: 'Returns current client version.',
+    params: [],
+    type: 'String'
+  },
+  sha3: {
+    aliasSection: 'web3',
+    description: 'Returns sha3 of the given data',
+    params: [{ name: '_', type: 'Bytes' }],
```

Comment:
```suggestion
    params: [{ name: 'data', type: 'Bytes' }],
```

Reply:
Just prefer this not being unnamed, since where we use TS (like in VSCode) it is nice when it shows the actual field in addition to the type. This also pulls through to things like the apps UI where these definitions are pulled-in, i.e. it allows you to see some descriptive name for the field 

(I've seen in frontier it has been slightly "lazy" where RPC params are unnamed, at least in the outer layers)

Reply:
I agree with you. I was just afraid it could somehow break code